homework/Avseenkov V/HomeWork8/Ex1_Animals/main.java

```diff
@@ -0,0 +1,13 @@
+import bean.Animal;
+import bean.Cat;
+import bean.Dog;
+
+public class main {
```

Comment:
Usually we name psvm as an Application.java with meaningful extend

we name class in Uppercase